lib/integrations/awsoidc/idp_iam_config.go

```diff
@@ -113,6 +144,14 @@ func (r *IdPIAMConfigureRequest) CheckAndSetDefaults() error {
 
 	r.ownershipTags = tags.DefaultResourceCreationTags(r.Cluster, r.IntegrationName)
 
+	switch r.IntegrationPolicyPreset {
+	case "":
+		r.IntegrationPolicyPreset = PolicyPresetUnspecified
```

Comment:
It's not clear to me why we need the `"unspecified"` instead of just using `""`?

Reply:
In this particular case, handling both empty string `""` and spelled out default `unspecified` value is not that important, I went ahead and removed it in https://github.com/gravitational/teleport/pull/47164/commits/2ee21fea32f04a1b38f16cacbe626a2906855297

To answer your question on when both empty string `""` and `unspecified` would be helpful - `""` and `unspecified` lets us distinguish set and unset value. Say when `tctl` ships with some constsants/enums in version `v1` and we support that feature in the web ui in `v2`. And since proxy and auth can have different versions, it can be tricky in auth to identify if web ui deliberately sent an empty string or is in a version that does not support the feature altogether.